dcs/src/main/resources/dcs-default.xml

```diff
@@ -386,4 +386,33 @@
         Timeout minutes between first and max times (6 default) DCS Server startup MXOSRVR.
     </description>
   </property>
+  <property>
+    <name>dcs.server.user.program.tcp.keepalive.status</name>
+    <value>enable</value>
+    <description>
+        Used in  mxosrvr keepalive , parameter is ENABLE IDLETIME INTERTIME RETRYCNT.
```

Comment:
Description should be specific to the property name/value..  specify the default for the property.  Look at how the description for statistics enable is done. This applies to the other tcp keepalive properties as well